git_lint_branch/linter_output.py

```diff
@@ -1,6 +1,6 @@
-from enum import Enum
+from enum import IntEnum
 
-class LinterLevel(Enum):
+class LinterLevel(IntEnum):
```

Comment:
This makes sense to do. Could you break it out into a different commit in the history, though? (Doesn't need to be a whole different branch, IMO.)

Reply:
Done. This is the one thing that our linter can't programmatically detect :p